lib/customs.js

```diff
@@ -22,7 +22,12 @@ module.exports = function (log, error) {
   function Customs(url) {
     if (url === 'none') {
       this.pool = {
-        post: function () { return P.resolve({ block: false })},
+        post: function (postUrl, postData) {
+          if (postUrl === '/check' && postData.action === 'accountLogin') {
```

Comment:
FYI this `url === 'none'` branch is used by e.g. self-hosters who don't have customs-server running.